src/node_file.cc

```diff
@@ -2619,6 +2680,7 @@ void Initialize(Local<Object> target,
   SetMethod(context, target, "access", Access);
   SetMethod(context, target, "close", Close);
   SetMethod(context, target, "open", Open);
+  SetMethod(context, target, "openFile", OpenFile);
```

Comment:
ditto here – it’s virtually impossible to tell what the difference between `open` and `openFile` is without looking at the implementations

Reply:
(unresolving this conversation since only the C++ name has been changed)